libs/application/templates/financial-aid/src/fields/TaxBreakdown/TaxBreakdown.css.ts

```diff
@@ -0,0 +1,60 @@
+import { style, globalStyle } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const tableContainer = style({
+  width: '100%',
+  backgroundColor: theme.color.white,
+  border: `1px solid ${theme.color.blue200}`,
+  '@media': {
+    [`print`]: {
+      marginBottom: theme.spacing[4],
+    },
+  },
+})
+
+globalStyle(`${tableContainer} th, ${tableContainer} td`, {
+  paddingRight: theme.spacing[3],
+  paddingLeft: theme.spacing[3],
+  paddingTop: theme.spacing[2],
+  paddingBottom: theme.spacing[2],
```

Comment:
Same question as before ⬆️

Reply:
We can, but i never remember the order of the padding 🙈  so i think this is more clearer for forgetful people like me 🤷🏼